sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -9,7 +9,7 @@
         <v-btn
           color="primary"
           class="mr-5"
-          v-if="!notDraft && !hasRestriction"
+          v-if="!notDraft && !hasRestrictio"
```

Comment:
There seems to be a typo here. Was the variable renamed to "hasRestrictio" ?

Reply:
yup its a typo